Add maxCharsPerColumn attribute to @CsvFileSource and @CsvSource (#2342)
junit-team/junit5: documentation/src/docs/asciidoc/release-notes/release-notes-5.7.0-M2.adoc

  resources.
* `JAVA_16` has been added to the `JRE` enum for use with JRE-based execution
  conditions.

> * `maxCharsPerColumn` attribute which enables the configuration of maximum characters
>   of per CSV column has been added to `@CsvSource` and `@CsvFileSource`.

Reviewer: ```suggestion
* `@CsvFileSource` and `@CsvSource` now provide a `maxCharsPerColumn` attribute
  that enables configuring the maximum number of characters per column.
```

Author: Updated, thanks for your suggestion.
